pkg/fftypes/event.go

```diff
@@ -69,9 +69,17 @@ type Event struct {
 // EnrichedEvent adds the referred object to an event
 type EnrichedEvent struct {
 	Event
-	Message         *Message         `json:"message,omitempty"`
-	Transaction     *Transaction     `json:"transaction,omitempty"`
-	BlockchainEvent *BlockchainEvent `json:"blockchainevent,omitempty"`
+	BlockchainEvent   *BlockchainEvent `json:"blockchainevent,omitempty"`
+	ContractAPI       *ContractAPI     `json:"contractAPI,omitempty"`
```

Comment:
Discussed that the API here is `/api/v1/namespaces/{ns}/apis`, but felt it was clearer to include `contractAPI` rather than just `api` here